dashboard/README.md

```diff
@@ -0,0 +1,38 @@
+# Court Transcript Dashboard
+
+## Viewing
+To view this dashboard, if not done so already, please make sure you have all the requirements installed. 
+This can be done by running `pip3 install -r requirements.txt` in the terminal; and is recommended to be done in a virtual environment.
+The dependencies include:
+```python
```

Comment:
maybe just add the commands for them to create the venv pip install the requirements in one line for ease.